blocks/profile-cards/profile-cards.css

```diff
@@ -114,17 +114,23 @@
 .profile-cards .card-1up .card-image-container,
 .profile-cards .card-1up-right .card-image-container {
   width: 100%;
-  border-radius: 10px 10px 0 0;
+}
+.profile-cards .card-1up-right .card-image-container img {
+  border-radius: 0px 10px 10px 0px;
```

Comment:
I think our ESlint would complain about these "0px"s. Could you make them all just "0"? Thanks!